app/src/common/sidebar/SidebarButton.module.scss

```diff
@@ -15,17 +19,24 @@
         font-family: var(--font-regular);
         user-select: none;
         font-size: 16px;
+        max-width: #{$caption-width}px;
+        overflow-x: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
     }
 
     &.indent-12 {
         :global(.uui-caption) {
-            margin-left: 12px;
+            margin-left: #{$index-1x}px;
+            max-width: #{$caption-width - $index-1x}px;
```

Comment:
If we set a max-width not at each button, but on the whole sidebar container, it will not work?

Reply:
done